same-optype/script.whizzml

```diff
@@ -0,0 +1,28 @@
+;; Given a source with incorrect optypes and a dataset with correct
```

Comment:
perhaps "with possibly incorrect"? (i presume the script will work even if the types are already fine)